### PR TITLE
fix configserver-lfd tests

### PR DESCRIPTION
**Pull request by mmetc**

### CODE REVIEW

File: .tests/csf-lfd-logs/config.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 parsers:
 - crowdsecurity/syslog-logs
-- crowdsecurity/configserver-lfd-logs.yaml
+- crowdsecurity/configserver-lfd-logs
```
Comment (AlteredCoder) [inline .tests/csf-lfd-logs/config.yaml:3]: i think it should be `crowdsecurity/s01-parse/configserver-lfd-logs.yaml`

**Review by AlteredCoder — CHANGES_REQUESTED** [review]

File: .tests/csf-lfd-logs/config.yaml (hunk 1 above)
Comment (rr404) [reply .tests/csf-lfd-logs/config.yaml]: sorry i had renammed csf-lfd-logs to configserver-lfd-logs also, so it also gonna mess the asserts

Comment (rr404) [reply .tests/csf-lfd-logs/config.yaml]: maybe

Comment (mmetc) [reply .tests/csf-lfd-logs/config.yaml]: yes I figured, thanks

**Pull request merged**